src/views/SettingsView.vue

```diff
@@ -1,23 +1,29 @@
 <template>
   <div>
     <navigation-header title="Settings"/>
-
     <div class="box">
-      Settings
+      <div class="level">
+        <label for="language-select" class="is-size-5 has-text-weight-bold level-left" v-html="'Language'"/>
+        <select class="level-right" id="language-select" v-model="$i18n.locale">
+          <option v-for="(lang, i) in langs" :key="`Lang${i}`" :value="lang">{{ lang }}</option>
+        </select>
+      </div>
     </div>
-
   </div>
 </template>
 
 <script>
 import NavigationHeader from '@/components/layout/NavigationHeader'
+import messages from '../utils/i18n'
```

Comment:
```suggestion
import messages from '@/utils/i18n'
```

You can use the @ sign instead